### PR TITLE
Fix: Added details about esc script to documentation

### PR DESCRIPTION
**Pull request by balloonpopper**

### CODE REVIEW

File: general/what_is_escoria.rst [hunk 1]
```diff
@@ -131,3 +131,16 @@ by Escoria. This way, the game developer has total control over the inventory.
...
+~~~~~~~~~~~
+
+Escoria uses scripts (a set of Escoria commands that include such actions as
+moving an object to a certain location, or playing a sound) to determine the
+behavior of the game world and the objects within it. ESC scripts are :
+
+* plain text files
+* created in any text editor (including the Godot script editor)
+* have the file extension ".esc"
+* can be stored in any location within your game's filesystem structure
```
Comment (StraToN) [inline general/what_is_escoria.rst:146]:
```suggestion
behavior of the game world and the objects within it. ESC scripts :

* are plain text files
* are created in any text editor (including the Godot script editor)
* have the file extension ".esc"
* can be stored in any location within your game's filesystem structure
```

File: general/what_is_escoria.rst [hunk 2]
```diff
@@ -131,3 +131,16 @@ by Escoria. This way, the game developer has total control over the inventory.
...
+
+ESC Scripts
+~~~~~~~~~~~
+
+Escoria uses scripts (a set of Escoria commands that include such actions as
+moving an object to a certain location, or playing a sound) to determine the
+behavior of the game world and the objects within it. ESC scripts are :
+
+* plain text files
+* created in any text editor (including the Godot script editor)
```
Comment (StraToN) [inline general/what_is_escoria.rst]:
```suggestion
behavior of the game world and the objects within it. ESC scripts :

* are plain text files
* are created in any text editor (including the Godot script editor)
```

**Review by StraToN — CHANGES_REQUESTED** [review]

File: getting_started/step_by_step/4_first_run.rst [hunk 3]
```diff
@@ -16,9 +16,10 @@ walk across the room, etc.) are very easy to use.
 
 The game start-up itself is also coordinated using an ESC script.
 
-Create a new ESC script which will be our main game script and call it
-"game.esc". Switch to the **Editor** tab and select "New Text File" from the
-"File" menu:
+Scripts are plain text files with a ".esc" extension, created in any text
```
Comment (dploeger) [inline getting_started/step_by_step/4_first_run.rst]:
```suggestion
Scripts are plain text files with an ".esc" extension, created in any text
```

File: general/what_is_escoria.rst (hunk 2 above)
Comment (dploeger) [reply general/what_is_escoria.rst]:
```suggestion
behavior of the game world and the objects within it. ESC scripts:

* are plain text files
* can be created in any text editor (including the Godot script editor)
```

Comment (dploeger) [reply general/what_is_escoria.rst]: Now you have two suggestions. 😄

**Review by dploeger — CHANGES_REQUESTED** [review]

File: general/what_is_escoria.rst (hunk 2 above)
Comment (StraToN) [reply general/what_is_escoria.rst]: Haha! Make up your mind :laughing: 
(@dploeger 's suggestion is better, though :D )

Comment (balloonpopper) [reply general/what_is_escoria.rst]: Sorry @StraToN , so far the score is @dploeger 1, @StraToN 0. :(

**Review by StraToN — APPROVED** [review]

**Comment by StraToN** [comment]
LGTM. Thanks a lot!

**Comment by StraToN** [comment]
LGTM. Thanks a lot!

File: getting_started/step_by_step/1_create_escoria_project.rst [hunk 4]
```diff
@@ -17,6 +17,24 @@ To make starting with Escoria as easy as possible, we provide a Godot Game
...
+mandated filesystem structure for your game. For example, you may
+choose to organise your files by :
+
+* purpose (e.g. "/items", "/scripts", "/sprites")
+* room (e.g. "/room_ballroom", "/room_study")
+* a combination of the two (e.g. "/room_ballroom/scripts",
+  "/room_ballroom/items" etc.)
+* some other structure
+
+You are not restricted to the structure this demonstration project uses, use
```
Comment (BHSDuncan) [inline getting_started/step_by_step/1_create_escoria_project.rst]:
```suggestion
You are not restricted to the structure used by this demonstration project: use
```

**Review by BHSDuncan — CHANGES_REQUESTED** [review]

**Review by BHSDuncan — APPROVED** [review]

**Review by StraToN — APPROVED** [review]

**Comment by StraToN** [comment]
Thanks ! 🥳

**Pull request merged**